CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ FetchContent_Declare(arcana
     GIT_TAG 10d167ffe0f86b2ddaccd93fd78ff2240cbb0c49)
 FetchContent_Declare(UrlLib
     GIT_REPOSITORY https://github.com/BabylonJS/UrlLib.git
-    GIT_TAG f09f431f24cac7ffb0f17e5a83ce6c2f11d29e31)
+    GIT_TAG d18aa71dd1a15f948344cf32059536e38e8cd935)
```

Comment:
git hash will change with a squash and merge of the PR. Or, to say it differently, a new commit will be pushed to urllib main branch when the PR will be merged. so, this commit number will not be valid.

Reply:
Ya, I will update it when the UrlLib PR is merged.